src/core/lib/iomgr/combiner.c

```diff
@@ -51,24 +51,53 @@ int grpc_combiner_trace = 0;
   } while (0)
 
 struct grpc_combiner {
+  grpc_combiner *next_combiner_on_this_exec_ctx;
   grpc_workqueue *optional_workqueue;
   gpr_mpscq queue;
   // state is:
   // lower bit - zero if orphaned
   // other bits - number of items queued on the lock
   gpr_atm state;
-  bool take_async_break_before_final_list;
+  // number of elements in the list that are covered by a poller: if >0, we can
+  // offload safely
+  gpr_atm covered_by_poller;
```

Comment:
please rename to elements_covered_by_poller to indicate that this is a counter and not a bool (as many other uses of the term covered_by_poller represent a bool)